docs/versioning/versioning-faq.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+nav_order: 3
+title: Versioning FAQ
+has_children: false
+parent: Versioning
+---
+
+# Versioning Frequently-asked-questions
```

Comment:
Maybe these could be moved to the main section of FAQs and just a link here to that page

Reply:
moved